sensor_msgs/include/sensor_msgs/point_field_conversion.h

```diff
@@ -80,6 +82,51 @@ namespace sensor_msgs{
   template<> struct typeAsPointFieldType<double>   { static const uint8_t value = sensor_msgs::PointField::FLOAT64; };
  
   /*!
+   * \Type names of the PointField data type.
+   */
+  int getPointFieldTypeFromString(const std::string field_name){
```

Comment:
Since this is not templated it needs to be inline with the implementation in the header. Also please switch to a const reference instead of pass by value of the string.